lib/phoenix_live_view/html_formatter.ex

```diff
@@ -50,6 +50,9 @@ defmodule Phoenix.LiveView.HTMLFormatter do
       ]
       ```
 
+    * `:migrate_eex_to_curl_brackets` - Automatically migrate single expression
```

Comment:
do we want to use curl or curly?

Reply:
Ah later curly is used, so it's just wrong in the docs 😃 
```suggestion
    * `:migrate_eex_to_curly_brackets` - Automatically migrate single expression
```